App.tsx

```diff
@@ -7,37 +7,56 @@ import GreetingScreen from './screens/GreetingScreen/GreetingScreen';
 import SignInScreen from './screens/SignInScreen/SignInScreen';
 import SignUpScreen from './screens/SignUpScreen/SignUpScreen';
 import HomeScreen from './screens/HomeScreen/HomeScreen';
-
+import { AuthContext } from './components/context';
 const Stack = createNativeStackNavigator();
 
 export default function App() {
-  // const [userToken, setUserToken] = useState('USERTOKEN');
-  const [userToken, setUserToken] = useState();
+  const [userToken, setUserToken] = useState('');
 
   const [isSignout, setSignout] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const authContext = React.useMemo(
```

Comment:
Por consistencia en el código, haría lo siguiente:
1. Cambiar el import en la primera linea a:
`import React, { useState, useMemo } from 'react';`
2.
```suggestion
  const authContext = useMemo(
```